lib/ejs.js

```diff
@@ -322,20 +323,21 @@ function includeSource(path, options) {
  * Re-throw the given `err` in context to the `str` of ejs, `filename`, and
  * `lineno`.
  *
- * @implements RethrowCallback
+ * @implements {RethrowCallback}
  * @memberof module:ejs-internal
  * @param {Error}  err      Error object
  * @param {String} str      EJS source
  * @param {String} filename file name of the EJS file
- * @param {String} lineno   line number of the error
+ * @param {Number} lineno   line number of the error
+ * @param {EscapeCallback} esc
  * @static
  */
 
-function rethrow(err, str, flnm, lineno, esc){
+function rethrow(err, str, filename, lineno, esc) {
   var lines = str.split('\n');
   var start = Math.max(lineno - 3, 0);
   var end = Math.min(lines.length, lineno + 3);
-  var filename = esc(flnm); // eslint-disable-line
+  filename = esc(filename);
```

Comment:
Really not keen on making changes to input params inline — is there not a better way to handle this